### DISCUSSION TITLE
Migrating from OpenAI to Azure: Managing Multiple Models

### DISCUSSION THREAD

**Discussion by Dreaming-Codes** [64bit/async-openai#295] [Q&A]
I'm trying to migrate my app from OpenAI endpoints to Azure ones, and I hoped it would be a simple configuration change. However, if I understand the Azure panel correctly, I can deploy only one model per deployment ID. This means I would need to use different clients for each model.
Am I missing something, or is that the intended way to use different models with Azure OpenAI deployments?

**Comment by 64bit** [comment]
`Client` reads the endpoint related information from a single`Config`, in this case `AzureConfig`, so one could have multiple configs to be used with single mutable client via `Client::with_config` or multiple clients with their own config.
